include/stl2/detail/concepts/object/move_constructible.hpp

```diff
@@ -25,6 +25,10 @@ STL2_OPEN_NAMESPACE {
 	template <class>
 	constexpr bool __addressable = false;
 	template <class T>
+	constexpr bool __addressable<T&> = true;
+	template <class T>
+	constexpr bool __addressable<T&&> = true;
```

Comment:
Oh, I misunderstood. You mean to leave the reference type specializations in for `__addressable`. Got it.